docs/map_state/embeddings.md

```diff
@@ -0,0 +1,24 @@
+Atlas stores, manages and generates embeddings for your unstructured data.
+
+You can access Atlas embeddings in their ambient form (e.g. high dimensional) or in their two-dimensional
+projected representations.
```

Comment:
embedded sounds confusing (map.embeddings.embedded?)

i am fine with either latent/ambient.


Another idea: what if we called them
two_dimensional
high_dimensional

instead?

Reply:
I'm lukewarm on `two_dimensional`/`high_dimensional`. Partly because I don't like writing out numbers.

My thought on 'embedded' is that accessing embeddings does just by default work on the latent space, and the two-d space is a special case. It's not like 'embeddings.vector_search' lets you search in the 2d space as an option.

So maybe like `map.embeddings.array`, `map.embeddings.ndarray`, `map.embeddings.matrix` just returns the latent space as an `n x d` np.ndarray, and `map.embeddings.projected` returns the `n x 2` array.